Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:slim
+
+COPY requirements.txt /requirements.txt
+
+RUN pip install --disable-pip-version-check --no-cache-dir -r /requirements.txt
+
+COPY . /tmp/bzdata
+
+WORKDIR /tmp/bzdata
```

Comment:
It would be cleaner to add a minimal `setup.py` to the project, and simply do a `python setup.py install` in the docker image, then the `bugzilla_dashboard` module will be available from anywhere (no need to work from a specific dir)